packages/graphql-transformer-common/src/__tests__/defnition.test.ts

```diff
@@ -31,28 +31,3 @@ describe('gets Non-Model Types', () => {
     expect((nonModelTypes[0] as ObjectTypeDefinitionNode).name?.value).toEqual('NonModel');
   });
 });
-
-describe('finds matching field', () => {
```

Comment:
Are these tests moved to a different package?

Reply:
Yes, you can find them here: https://github.com/aws-amplify/amplify-category-api/blob/cb370f2fbcbbedc077483d9b66414ea2119577be/packages/amplify-graphql-schema-generator/src/__tests__/schema-overrides.test.ts#L600